src/lib/stitching/gravity/schema.ts

```diff
@@ -49,53 +48,49 @@ export const executableGravitySchema = () => {
     "ArtistSeries",
     "ArtistSeriesEdge",
     "ArtistSeriesConnection",
+    "SecondFactor",
```

Comment:
Once the gravity backend schema has been cleaned up and updated in MP, we can remove mention of these fields (and the mutations below) from the blocklist (as they will no longer even exist in Gravity).